extra/mariabackup/xtrabackup.cc

```diff
@@ -1816,6 +1816,13 @@ static void print_version(void)
       my_progname, MYSQL_SERVER_VERSION, SYSTEM_TYPE, MACHINE_TYPE);
 }
 
+static void concatenate_default_groups(std::vector<const char*> &backup_load_groups, const char **default_groups)
+{
+	for ( ; *default_groups ; default_groups++) {
```

Comment:
We have coding standards (see https://github.com/MariaDB/server/blob/11.0/CODING_STANDARDS.md), as stated in PR template.
`{` should go in the new line, but since you have single command, no need for curly braces.

Reply:
Hi Anel, Thank you for your guidance. I will make the necessary modifications to comply with the coding standards.